### PR TITLE
feat(core): Add type and configuration file for MemberManagement

### PR DESCRIPTION
**Pull request by rakeshsinha31**
The typedef is not yet added in config file.

### CODE REVIEW

File: src/types/member-management.d.ts [hunk 1]
```diff
@@ -1,4 +1,71 @@
 import { ID } from './types';
+import { Connection } from './connection';
+import { DateTimeFilter } from './filter';
+
+export enum MemberStatus {
```
Comment (rogermadjos) [inline src/types/member-management.d.ts]: or `enums`, the convention is, the enum values should be in `PascalCase`, e.g.
`Enabled`
`Disabled`
`Web`

File: src/types/member-management.d.ts [hunk 2]
```diff
@@ -1,4 +1,71 @@
...
+export type MemberParams = {
+  admin: ID;
+  first?: number;
+  after?: string;
+  filter?: {
+    dateOfBirth?: DateTimeFilter;
+  };
+};
+
+export type Member = {
```
Comment (rogermadjos) [inline src/types/member-management.d.ts]: you forgot to add `id`

File: src/types/member-management.d.ts [hunk 3]
```diff
@@ -1,4 +1,71 @@
...
+  dateTimeCreated: string;
+  dateOfBirth: string;
+  tags: [string];
```
Comment (rogermadjos) [inline src/types/member-management.d.ts]: this should be `string[]`

File: src/types/member-management.d.ts [hunk 4]
```diff
@@ -18,4 +85,8 @@ export type MemberManagemnetAPI = {
     status: string,
```
Comment (rogermadjos) [inline src/types/member-management.d.ts]: best to use the `MemberStatus` type you defined earlier

File: src/types/member-management.d.ts (hunk 1 above)
Comment (rakeshsinha31) [reply src/types/member-management.d.ts]: done

File: src/types/member-management.d.ts (hunk 3 above)
Comment (rakeshsinha31) [reply src/types/member-management.d.ts]: done

**Comment by rakeshsinha31** [comment]
change request has been implemented.

**Pull request closed without merging**